keywording/text.xml

```diff
@@ -421,14 +421,53 @@ dependency for your package).
 </p>
 
 <p>
-Some architectures (<c>alpha</c>, <c>ia64</c>, <c>mips</c>, <c>riscv</c>,
+Some architectures (<c>alpha</c>, <c>loong</c>, <c>mips</c>, <c>riscv</c>,
 <c>s390</c>) do not maintain a stable keyword, so packages are not to be marked
 stable for one of these architectures.
 </p>
 
 </body>
 </subsection>
 
+<subsection>
+<title>Stabilization groups</title>
+<body>
+
+<p>
+Stabilization groups are sets of packages defined to help organize and automate
+stabilization requests. These groups are stored under
+<c>metadata/stabilization-groups/</c>, and all files within this directory
+(including subdirectories) are included recursively.
+</p>
+
+<p>
+Each group is named according to its path, for example <c>@gnome/vte</c>. This
+naming convention helps avoid collisions and makes referencing groups
+straightforward. The files use a simple format: one <c>cat/pkg</c> per line,
+with lines starting with <c>#</c> treated as comments. Whitespace before and
+after entries is ignored.
+</p>
+
+<p>
+When invoking tools like <c>pkgdev bugs</c>, passing a set (anything starting
+with <c>@</c>) will expand it to all packages listed in that set. For example,
+<c>@gnome/vala</c> would expand to <c>dev-libs/vala-common</c> and
+<c>dev-lang/vala</c>.
+</p>
+
+<p>
+During bug creation, packages whose latest matching version is already
+stabilized are dropped from the list. The tool then builds the full dependency
+graph as usual, adding any additional required packages. For each defined set,
+all packages included in both the graph and the set are merged into a single
+bug. For example, if <c>@gnome/vala</c> contains <c>dev-libs/vala-common</c> and
+<c>dev-lang/vala</c>, and both are required, they will be grouped into one bug.
+Packages in the set that are not required or requested are not included.
```

Comment:
How can a package in the set be "not requested"? I don't understand that part.

Reply:
@sbraz so the usage of stabilization groups is two fold:

1. expand mentions of `@set` into all packages "as requested". But you can also manually request packages, including subset of the group.
2. force-group packages from the same group into one bug, so they are done together.

Reply:
Yes, https://pkgcore.github.io/pkgdev/man/pkgdev/bugs.html